packages/components/src/Menu/Menu.css

```diff
@@ -80,8 +80,23 @@
   padding: var(--space-small);
 }
 
+:global(.jobber-retheme) .sectionHeader {
+  display: flex;
+  align-items: center;
+  min-height: var(--space-largest);
+  padding: var(--space-small) var(--space-base);
+}
+
+:global(.jobber-retheme) .sectionHeader h6 {
+  /* can move into the Typography component post re-theme */
+  font-size: var(--typography--fontSize-base);
+  font-weight: 600;
```

Comment:
Sorry forgot to update this in Figma, this remained un-bold or 400 weight

Reply:
I don't think it did? 
![image](https://github.com/GetJobber/atlantis/assets/39704901/36f55d25-d5cc-403e-b704-956718ae7c17)

Reply:
Ohhh right this is the section header! On it